### PR TITLE
updates to allow spacing in the showcase

### PR DESCRIPTION
**Pull request by jonhartzler**
updates

### CODE REVIEW

File: app/views/sections/_image_form.html.erb [hunk 1]
```diff
@@ -1,6 +1,10 @@
 <div class="row">
   <div class="col-md-9">
     <%= f.input :caption, input_html: { class: 'honeycomb_redactor' }, as: :text %>
+
+    <%= f.input :has_spacer, as: :boolean, label: false, hint: "Adds space in front of this section." do %>
```
Comment (jgondron) [inline app/views/sections/_image_form.html.erb]: Should it read "Add space in front of this section."?

Comment (jonhartzler) [inline app/views/sections/_image_form.html.erb]: sounds good to me

**Comment by jgondron** [comment]
:+1:

**Comment by danthewolfe** [comment]
:+1:

**Pull request merged**